stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -598,6 +598,11 @@ internal class ChatDomainImpl internal constructor(
         // 0 ensure load is complete
         initJob.join()
 
+        activeChannelMapImpl
+            .values
+            .filter { it.errorWatching }
+            .forEach { it.watch() }
```

Comment:
This might be worth a comment similar to the other ones in this method

Reply:
:+1: